content/next/index.md

```diff
@@ -71,39 +73,49 @@ docs: correct spelling of CHANGELOG
+### Commit message with multi-paragraph body and multiple footers
 ```
 fix: correct minor typos in code
 
-see the issue for details on the typos fixed
+see the issue for details
 
-closes issue #12
+on typos fixed.
+
+Reviewed-by: Z
+Refs: #133
 ```
 
 ## Specification
 
 The key words “MUST”, “MUST NOT”, “REQUIRED”, “SHALL”, “SHALL NOT”, “SHOULD”, “SHOULD NOT”, “RECOMMENDED”, “MAY”, and “OPTIONAL” in this document are to be interpreted as described in [RFC 2119](https://www.ietf.org/rfc/rfc2119.txt).
 
 1. Commits MUST be prefixed with a type, which consists of a noun, `feat`, `fix`, etc., followed
-  by an OPTIONAL scope, and a REQUIRED terminal colon and space.
+  by the OPTIONAL scope, OPTIONAL `!`, and REQUIRED terminal colon and space.
 1. The type `feat` MUST be used when a commit adds a new feature to your application or library.
 1. The type `fix` MUST be used when a commit represents a bug fix for your application.
 1. A scope MAY be provided after a type. A scope MUST consist of a noun describing a
   section of the codebase surrounded by parenthesis, e.g., `fix(parser):`
-1. A description MUST immediately follow the space after the type/scope prefix.
+1. A description MUST immediately follow the colon and space after the type/scope prefix.
 The description is a short summary of the code changes, e.g., _fix: array parsing issue when multiple spaces were contained in string._
 1. A longer commit body MAY be provided after the short description, providing additional contextual information about the code changes. The body MUST begin one blank line after the description.
-1. A footer of one or more lines MAY be provided one blank line after the body. The footer MUST contain meta-information
-about the commit, e.g., related pull-requests, reviewers, breaking changes, with one piece of meta-information
-per-line.
-1. Breaking changes MUST be indicated in the type/scope prefix of a commit, or at the beginning of its optional body or footer section.
-1. If included in the body or footer, a breaking change MUST consist of the uppercase text BREAKING CHANGE, followed by a colon, space, and description, e.g.,
+1. A commit body is free-form and MAY consist of any number of newline separated paragraphs.
+1. One or more footer lines MAY be provided one blank line after the body. Each footer MUST
+ follow the [git trailer convention](https://git-scm.com/docs/git-interpret-trailers): consisting of
+ a word token, a colon and space, followed by a string value.
+1. A footer's token MUST use `-` in place of whitespace characters, e.g., `Acked-by` (this helps differentiate
+  the footer section from a multi-paragraph body). An exception is made for `BREAKING CHANGE`, which MAY also be used as a token.
+1. A footer's value MAY contain spaces and newlines, but MUST follow the folding conventions described
+  in [RFC 822](https://www.ietf.org/rfc/rfc822.txt) (each newline MUST have a leading space).
```

Comment:
I'd vote to make the leading space on newlines optional.

There are three reasons:

1. it makes it easier to write commits in a natural way, and still adhere to the spec and parsers can pick up your multi-line trailers.
2. if someone has their editor configured to strip trailing whitespaces, then this makes it much more difficult to write multi-paragraph trailers, because a blank line with a single whitespace to make it (and the next line) still part of the existing trailer might get stripped when saving the commit, unless you remember to disable your formatter so it won't remove the trailing whitespace
3. it makes it ambiguous what to do if someone _does not_ put a space in front of the new lines, but is actually writing a paragraph after a previous trailer. Is this section now part of the body again, is it simply ignored by parsers, or ...?

Given the above three negatives from requiring the space, and given the fact that the parser I wrote [already shows that it's possible to implement a non-ambiguous parser](https://docs.rs/conventional/0.4.1/conventional/#example) for the spec which doesn't require the extra space on a newline, I'd say there are clear advantages to removing this requirement.